synapse/lib/nexus.py

```diff
@@ -463,6 +467,11 @@ async def runMirrorLoop(self, proxy):
                 genr = proxy.getNexusChanges(offs, **opts)
                 async for item in genr:
 
+                    if self.readonly:
+                        logger.error('Unable to consume Nexus events when readonly is set')
+                        await self.client.fini()
```

Comment:
This will not automatically resume if the cell resumes normal operation. May need a sleep loop rather than return and logger to say it's re-enabled.

Reply:
The cell free space loop calls `nexsroot.startup()` when resuming normal operation which restarts this telepath client and loop.